pkg/experiment/query_backend/query_profile_entry.go

```diff
@@ -72,7 +73,12 @@ func getSeriesLabels(reader phlaredb.IndexReader, matchers []*labels.Matcher, by
 	series := make(map[uint32]seriesLabels)
 	l := make(phlaremodel.Labels, 0, 6)
 	for postings.Next() {
-		fp, err := reader.SeriesBy(postings.At(), &l, &chunks, by...)
+		var fp uint64
+		if len(by) == 0 {
+			fp, err = reader.Series(postings.At(), &l, &chunks)
+		} else {
+			fp, err = reader.SeriesBy(postings.At(), &l, &chunks, by...)
```

Comment:
This function was deleting my labels and was keeping me from fetching them, so I used `Series` instead when`by` is empty.